src/__tests__/components/forms/SubmitTemplate/SubmitTemplate.test.tsx

```diff
@@ -5,6 +5,10 @@ jest.mock('@utils/amplify-utils', () => ({
   getAmplifyBackendClient: () => {},
 }));
 
+jest.mock('@forms/SubmitTemplate/server-action', () => ({
+  submitTemplate: 'submitTemplatePlaceholder',
+}));
```

Comment:
FYI - I had to remove in my latest PR since in the sms and email submit we are using the bind function on submitTemplate. The test will complain about it being undefined.

Reply:
So long as you've not re-introduced the error logged during unit tests.

Reply:
yeah it does 😢 I'll take a look into it